packages/paste-website/src/pages/customization/customization-provider.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: Customization Provider
+description: The Customization Provider allows a consumer to customize the style of Paste components.
+slug: /customization/customization-provider/
+---
+
+import {graphql} from 'gatsby';
+import PackageJSON from '@twilio-paste/customization/package.json';
+import Changelog from '@twilio-paste/customization/CHANGELOG.md';
+import {ThemeObjectDisplay} from '../../components/ThemeObjectDisplay';
+import {ProgrammableAppsCallout} from '../../components/page-components/customization/customization-provider/ProgrammableAppsCallout';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/customization/customization-provider/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.CUSTOMIZATION}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-customization"
+  storybookUrl="?path=/story/theme-customization--provider"
+  data={[
+    {
+      node: {
+        name: PackageJSON.name,
+        description: PackageJSON.description,
+        version: PackageJSON.version,
+      },
+    },
+  ]}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+The `CustomizationProvider` provides a central place for customizations to be applied to an application that is using Paste.
+
+Say you want to build a web application or website using Paste, but want to have your own look and feel—the `CustomizationProvider` exists to do just that.
+
+If you&rsquo;re a Flex customer, and want to build a Flex plugin that follows your brand guidelines, the `CustomizationProvider` can be used for that as well.
+
+<ProgrammableAppsCallout />
+
+## About the Customization Provider
+
+Similar to the Paste [ThemeProvider](/tokens/theme-package), the `CustomizationProvider` uses [React Context](https://reactjs.org/docs/context.html) to provide a [theme object](/tokens/theme-package/#theme) to any descendant components in the tree.
+
+The `CustomizationProvider` currently differs in two ways:
+
+1. Firstly, you set a `baseTheme`. The base theme tells the provider which Paste theme you would like to base your custom theme on. You can choose from the 'Default' or 'Dark' Theme.
+2. Secondly, you set a `theme`, but unlike with the `<Theme.Provider />` you pass it an object.
```

Comment:
Have you considered keeping `theme` the same and adding a `customizations` prop instead?

Reply:
I hadn't entirely, but it was starting to bug me. I think it should be theme related as that's what it is, and `customizations` might not be explicit enough. But as this prop is a partial and merged into the base theme, `theme` is also not telling the the full story

`customTheme`? `themeOverrides`?

Reply:
`overrides` sounds good

Reply:
Another data point. Both [material](https://material-ui.com/customization/theming/) and [chakra](https://chakra-ui.com/docs/theming/customize-theme) use `theme` but provide a helper function to merge overrides into a base theme. 

I wonder if there's any merit in keeping that API similar for transferability? Is there value in a helper for creating a theme and not overloading the provider